src/pocket.jl

```diff
@@ -0,0 +1,76 @@
+function itpos2coord(idxpos, tmitps)
+    return [itp(idxpos) for itp in tmitps]
+end
+
+function z2itpos(z, itpz)
```

Comment:
This could definitely be more clear.

The idea was to make it easy to get the coordinates of the splines at an arbitrary z coordinate. So the process is:
  z coordinate -> interpolation index (from 1 to the length of the TM) -> 3D interpolated position

Good call on specifying the type of `itpz`!